src/Compilers/CSharp/Portable/Errors/MessageProvider.cs

```diff
@@ -22,72 +22,33 @@ void IObjectWritable.WriteTo(ObjectWriter writer)
             // write nothing, always read/deserialized as global Instance
         }
 
-        Func<ObjectReader, object> IObjectReadable.GetReader()
```

Comment:
Please revert all stylistic changes in this file. They are not appropriate for stabilization fixes.